source/administration/console/security-and-access.rst

```diff
@@ -0,0 +1,157 @@
+
+
+.. _minio-console-security-access:
+
+===================
+Security and Access
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+You can use the MinIO Console to perform several of the identity and access management functions available in MinIO.
```

Comment:
This is actually a good pattern. Suggest applying to the other pages so the lead-in remains consistent throughout. Maybe add a ', such as:' to tie into the list.

Reply:
Done